src/index.css

```diff
@@ -12,3 +38,46 @@ body {
 	margin: 2rem auto;
 	max-width: 35rem;
 }
+
+a,
+a:visited {
+	color: var(--link-color);
+}
+
+input {
+	background: var(--bg-color);
+}
+
+input[type="text"],
+input[type="email"],
+input[type="password"] {
+	border: 1px solid hsl(0, 0%, 60%);
+	border-radius: 6px;
+	vertical-align: middle;
+}
+
+.ThemeToggleButton {
+	align-self: center;
+	width: 50px;
+	height: 24px;
+	border-radius: 12px;
+	border: 0;
+	background-color: transparent;
+	background-image: url('components/theme-toggle-button.png');
+	background-size: 76px 24px;
+	background-position: -26px 0;
+	transition: 100ms;
+	text-indent: -999px;
+	overflow: hidden;
+	opacity: 0.8;
+}
+.ThemeToggleButton:focus {
+	outline: none;
+}
+.ThemeToggleButton:hover {
+	opacity: 1;
+	cursor: pointer;
+}
+.dark-theme .ThemeToggleButton {
```

Comment:
conflicting css/html naming convention?
what do we go for in a react context?

Reply:
for the `.dark-theme`? Yea not sure. Utility? So `.is-dark` maybe?

Reply:
no clue!
I'd say a classic BEM? with emphasis on components, and matching the naming of css/html to the naming of react?

Reply:
i only know the suit css ones from back in the day but I follow you here!